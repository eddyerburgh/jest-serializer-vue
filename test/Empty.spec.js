import { mount, shallow } from '@vue/test-utils'
import Empty from './components/Empty.vue'

describe('Empty.vue', () => {
  it('mount snapshot', () => {
    const wrapper = mount(Empty)
    expect(wrapper.html()).toMatchSnapshot()
    expect(wrapper.element).toMatchSnapshot()
  })

  it('shallow snapshot', () => {
    const wrapper = shallow(Empty)
    expect(wrapper.html()).toMatchSnapshot()
  })

  it('properly serializes a shallowly-rendered wrapper', () => {
    const wrapper = shallow(Empty)
    expect(wrapper).toMatchSnapshot()
  })

  it('properly serializes a fully-mounted wrapper', () => {
    const wrapper = mount(Empty)
    expect(wrapper).toMatchSnapshot()
  })
})
