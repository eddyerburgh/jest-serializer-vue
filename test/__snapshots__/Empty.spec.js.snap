// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty.vue mount snapshot 1`] = `undefined`;

exports[`Empty.vue mount snapshot 2`] = `<!---->`;

exports[`Empty.vue shallow snapshot 1`] = `undefined`;
